core/sql/bin/ex_sscp_main.cpp

```diff
@@ -248,12 +248,13 @@ void runServer(Int32 argc, char **argv)
   {
      cout << tmbuf << " RMS Shared segment exists, attaching to it, shmid="<< shmid << ", key=" << (key_t)getStatsSegmentId() << "\n";
   }
-  if ((statsGlobalsAddr = shmat(shmid, getRmsSharedMemoryAddr(), 0))
+  if ((statsGlobalsAddr = shmat(shmid, getRmsSharedMemoryAddr(), SHM_REMAP | SHM_RND))
```

Comment:
This shared segment can't be remapped because RMS shared segment have pointers stored within its space. All the processes attaching to the shared segment should attach at the same address. If the process is unable to do it, then the process can't participate in updating the shared segment. This can lead to corruption in the shared segment